src/components/card-list/card-list.component.jsx

```diff
@@ -5,17 +5,17 @@ import Card from '../card/card.component';
 import BackToTop from '../back-to-top/back-to-top.component';
 
 const CardList = ({ pokemon }) => (
-  <div>
```

Comment:
no need for outer `<div>` which does nothing

Reply:
okie dokie